EnglishFilter/sections/css_extended.txt

```diff
@@ -540,7 +543,7 @@ oddsdigest.com#?#.td-ss-main-sidebar > .td_block_template_1:has(> h4 > span:cont
 reblop.com#?#.td-block-row > div[class^="td-block-span"]:has(a[href^="https://t.frtyo.com/"])
 ! https://github.com/AdguardTeam/AdguardFilters/issues/33446
 duplichecker.com#$?#.adsbygoogle { remove: true; }
-duplichecker.com#?#.side-bar > .search-panel-sec:has(> .adds_box)
+.com#?#.side-bar > .search-panel-sec:has(> .adds_box)
```

Comment:
It seems that there is something wrong with this rule :)

By the way, they display different ads if they detect ad blocker, so I think that we could also change this rule:
https://github.com/AdguardTeam/AdguardFilters/blob/9c93e652efbf054fc788271ed86cd8ce951595f6/EnglishFilter/sections/antiadblock.txt#L1071

To for example something like:
`duplichecker.com#%#//scriptlet("set-constant", "google_jobrunner", "falseFunc")`
or/and also add this:
`duplichecker.com#%#//scriptlet("prevent-setInterval", "isAdBlockActive")`

Reply:
@AdamWr for `different ads if they detect ad blocker` this is ok `duplichecker.com#%#//scriptlet("prevent-setInterval", "isAdBlockActive")`
These rules should be enough for content blockers
```
duplichecker.com#@#.adsbygoogle
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=duplichecker.com
@@||pagead2.googlesyndication.com/pagead/js/*/show_ads_impl*.js$domain=duplichecker.com
```